src/components/Wizard/NewEditImportPane/__tests__/ExperimentImportMessageForm.test.tsx

```diff
@@ -0,0 +1,489 @@
+/*
+ * Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/
+ */
+
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  test,
+  jest,
+} from "@jest/globals";
+import { types } from "@mozilla/nimbus-shared";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ReactNode } from "react";
+import selectEvent from "react-select-event";
+
+import ExperimentImportMessageForm from "../ExperimentImportMessageForm";
+import Toasts from "../../../Toasts";
+import useToasts from "../../../../hooks/useToasts";
+import { MessageTemplate } from "../../messageTypes";
+import WizardFormData from "../../formData";
+
+type NimbusExperiment = types.experiments.NimbusExperiment;
+type Branches = NimbusExperiment["branches"];
+
+function recipe(slug: string, branches?: Branches): NimbusExperiment {
+  const featureIds = new Set<string>();
+  for (const branch of branches ?? []) {
+    const features =
+      "features" in branch ? branch["features"] : [branch["feature"]];
+    for (const feature of features) {
+      featureIds.add(feature.featureId);
+    }
+  }
+
+  return {
+    slug,
+    id: slug,
```

Comment:
I see `[content.id]: Field was not deserialized` showing up for Spotlights; but that branch ID is what's being used as message ID in the form - so that error message is probably not accurate. I don't know how big of a lift it would be to exempt just that field from the errors, maybe a cleanup for later?

Reply:
As we discussed, `content.id` is not the same as `id`. We can handle this case when we support arbitrary fields.